.github/workflows/test.yml

```diff
@@ -28,12 +28,32 @@ jobs:
   test-test262:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
-    - name: use node.js v16.x
-      uses: actions/setup-node@v1
-      with:
-        node-version: 16.x
-    - run: npm ci
-    - run: npm run test262
-      env:
-        HEAD_SHA: ${{ github.event.pull_request.head.sha }}
+      - uses: actions/checkout@v2
+      - name: use node.js v16.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 16.x
+      - run: npm ci
+      - run: npm run test262
+        env:
+          HEAD_SHA: ${{ github.event.pull_request.head.sha }}
+  test-lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: use node.js v16.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 16.x
+      - run: npm ci --no-optional
```

Comment:
This call (and the one below) use `--no-optional`, but the test262 one doesn't - why?

Reply:
No idea. I just copied the section from lint. Perhaps because optional dependencies don't impact linting... and won't impact ts-prune either. So probably just a way to save on build time and cloud cost. In any case, changing this is probably out of scope for this PR. If you think it's a good setting to do elsewhere, feel free to open an issue and we can discuss!

Reply:
BTW, per https://github.com/js-temporal/temporal-polyfill/pull/62#discussion_r719647627 I put the prune step in the wrong YML file. It's fixed now.